src/styles/mixins.scss

```diff
@@ -0,0 +1,6 @@
+@mixin font-fam($font-fam, $text-size, $text-weight, $line-height) {
```

Comment:
The parameter `$font-fam` is defined but not used in the mixin. If you intend to allow dynamic font families, you should use `$font-fam` in the `font-family` property.